README.md

```diff
@@ -62,88 +65,25 @@ $ pushd $CLOUDH
 $ sudo setcap cap_net_admin+ep ./cloud-hypervisor/target/release/cloud-hypervisor
 $ ./cloud-hypervisor/target/release/cloud-hypervisor \
 	--kernel ./target/target/release/hypervisor-fw \
-	--disk path=./clear-29160-kvm.img \
+	--disk path=focal-server-cloudimg-amd64.raw \
 	--cpus boot=4 \
 	--memory size=512M \
 	--net "tap=,mac=,ip=,mask=" \
 	--rng
 $ popd
 ```
 
-### Firecracker
+### QEMU
 
-As per [quick
-start](https://github.com/firecracker-microvm/firecracker/blob/master/docs/getting-started.md)
+Use the QEMU `-kernel` parameter to specify the path to the firmware.
 
-Replacing the kernel and rootfs to point at the firmware and the full disk
-image instead.
+e.g.
 
 ```
-curl --unix-socket /tmp/firecracker.socket -i \
-    -X PUT 'http://localhost/boot-source'   \
-    -H 'Accept: application/json'           \
-    -H 'Content-Type: application/json'     \
-    -d '{
-        "kernel_image_path": "target/target/release/hypervisor-fw",
-        "boot_args": ""
-    }'
-
-curl --unix-socket /tmp/firecracker.socket -i \
-    -X PUT 'http://localhost/drives/rootfs' \
-    -H 'Accept: application/json'           \
-    -H 'Content-Type: application/json'     \
-    -d '{
-        "drive_id": "rootfs",
-        "path_on_host": "clear-28660-kvm.img",
-        "is_root_device": true,
-        "is_read_only": false
-    }'
-
-curl --unix-socket /tmp/firecracker.socket -i \
-    -X PUT 'http://localhost/actions'       \
-    -H  'Accept: application/json'          \
-    -H  'Content-Type: application/json'    \
-    -d '{
-        "action_type": "InstanceStart"
-     }'
-
+$ qemu-system-x86_64 -machine q35,accel=kvm -cpu host,-vmx -m 1G\
+    -kernel ./target/target/release/hypervisor-fw \
+    -display none -nodefaults \
+    -serial stdio \
+    -drive id=os,file=focal-server-cloudimg-amd64.raw,if=none \
+    -device virtio-blk-pci,drive=os,disable-legacy=on
 ```
-
-**Currently Firecracker's virtio block device does not support resetting the
-device and as such it is not possible for the booted Linux kernel to take over
-the device from the firmware.**
-
-## Testing
-
-"cargo test" needs disk images from make-test-disks.sh
-
-And clear-28660-kvm.img:
-
-https://download.clearlinux.org/releases/28660/clear/clear-28660-kvm.img.xz
-
-sha1sum: 5fc086643dea4b20c59a795a262e0d2400fab15f
-
```

Comment:
This section is still valid because `cargo test` needs FAT disk images generated from `make-test-disks.sh` and `clear-28660-kvm.img`. Does this deletion is intended to hide the test from the README?